tox.ini

```
@@ -1,6 +1,6 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = py27-django{18,111},py36-django{18,111}
+envlist = py27-django{111,22},py36-django{111,22}
```

py27とdjango-2.2の組合せはテスト不要だと思います。

https://www.djangoproject.com/download/
> The last version to support Python 2.7 is Django 1.11 LTS.

```suggestion
envlist = py27-django111,py36-django{111,22}
```